gapic-common/lib/gapic/rest/client_stub.rb

```diff
@@ -174,12 +180,13 @@ def make_http_request verb, uri:, body:, params:, options:, is_server_streaming:
       # @param is_server_streaming [Boolean] flag if method is streaming
       # @yieldparam chunk [String] The chunk of data received during server streaming.
       # @return [Faraday::Response]
-      def base_make_http_request verb, uri:, body:, params:, metadata:, timeout:, is_server_streaming: false
-        if @numeric_enums && (!params.key?("$alt") || params["$alt"] == "json")
+      def self.base_make_http_request connection:, verb:, uri:, body:, params:, metadata:,
```

Comment:
Is there a reason this was moved to the class object? I thought it was fine where it was.

Reply:
easier to stub when testing

Reply:
...
no reason
...
...
(for some reason I thought it would make testing easier)